docs/topics/wasm/wasm-overview.md

```diff
@@ -50,6 +48,67 @@ With Compose Multiplatform for Web, you can run your code in the browser with al
 * [Get started with Kotlin/Wasm in IntelliJ IDEA](wasm-get-started.md)
 * Check out the [GitHub repository with Kotlin/Wasm examples](https://github.com/Kotlin/kotlin-wasm-examples)
 
+## Libraries support
+
+You can use the Kotlin standard library (`stdlib`) and test library ([`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/))
+in Kotlin/Wasm out of the box. The version of these libraries is the same as the version of the `kotlin-multiplatform` plugin.
+
+Other official Kotlin (`kotlinx`) and multiplatform libraries are not fully supported yet. You can try experimental versions of such libraries
+by adding the Kotlin [experimental repository](https://maven.pkg.jetbrains.space/kotlin/p/wasm/experimental/) to your Gradle project.
+
+> For Kotlin 1.9.0-Beta and later, use the latest available libraries' versions.
+> 
+{type="note"}
+
+### Enable kotlinx libraries
+
+To set a dependency on a Kotlin library (`kotlinx`), such as [`kotlinx.serilization`](serialization.md) and [`kotlinx.coroutines`](coroutines-guide.md),
+update your `build.gradle.kts` file:
+
+```kotlin
+// `build.gradle.kts`
+
+repositories {
+    maven {
+        url = uri("https://maven.pkg.jetbrains.space/kotlin/p/wasm/experimental")
+    }
+}
+
+kotlin {
+    sourceSets {
+        val wasmMain by getting {
+            dependencies {
+                implementation("org.jetbrains.kotlinx:kotlinx-serialization-core-wasm:1.5.1-wasm0")
```

Comment:
The text above mention `kotlinx.coroutines`, could be better to add it here then.

Reply:
Added